lib/zaikio/jwt_auth.rb

```diff
@@ -119,7 +119,9 @@ def authenticate_by_jwt
       end
 
       def update_revoked_access_tokens_by_webhook
-        return unless params[:name] == "directory.revoked_access_token"
+        unless params[:name] == "directory.revoked_access_token" || params[:name] == "zaikio.revoked_access_token"
+          return
+        end
```

Comment:
Maybe use an array and `include?` to improve the readability slightly?

```suggestion
        return unless %w[directory.revoked_access_token zaikio.revoked_access_token].include?(params[:name])
```